builders/interact/ethereum-api/wallets/metamask.md

```diff
@@ -72,7 +72,7 @@ Here, you can configure MetaMask for the following networks:
 
 |         Variable          |                           Value                            |
 |:-------------------------:|:----------------------------------------------------------:|
-|       Network Name        |               `EVM ContainerChain Dancebox`                |
+|       Network Name        |               `EVM Appchain Dancebox`                |
```

Comment:
can you please make sure this table is formatted?

Reply:
done